packages/renderer/src/lib/markdown/micromark-button-directive.ts

```diff
@@ -40,8 +40,19 @@ export function button(d: Directive): void {
     return false;
   }
 
-  // Make sure {command=example} has been passed in
-  if (d.attributes && 'command' in d.attributes) {
+  // If {command=example args='["arg1"]'} has been passed in
+  // Icon format of a button
+  if (d.attributes && 'command' in d.attributes && 'args' in d.attributes) {
+    this.tag(`
+      <button 
+        class="fa-solid ${this.encode(d.label)} before:px-1 fa-3x" 
```

Comment:
```suggestion
        class="fa-solid ${this.encode(d.label)} before:px-1 fa-3x hover:text-[var(--pd-action-button-primary-hover-text)]" 
```

let's add some colors on hovering 🌈 (currently feel not really clickable)

https://github.com/user-attachments/assets/034496e2-677d-4d7f-867e-97bc9cc495cd


https://github.com/user-attachments/assets/e222827b-e5cd-442f-931b-0ee14f7a87be